gstd/gstd_bus_msg.c

```diff
@@ -136,6 +137,7 @@ gstd_bus_msg_to_string (GstdObject * object, gchar ** outstring)
   GstMessage *target;
   gchar *ts;
   GValue value = G_VALUE_INIT;
+  GstdIFormatter *formatter = g_object_new (GSTD_TYPE_JSON_BUILDER, NULL);
```

Comment:
Do you feel this will have a lower performance toll than locking the access to the builder?

Reply:
In the regular use case, I think that creating a new object has more overhead than waiting for a mutex. But in the regular use case having an overhead is not that important.
On the other hand in the extreme case (multiple threads doing continuous requests) waiting for the mutex will be a bottleneck and in that case, the overhead is actually important. 
I prefer maintaining a constant overhead on the calls instead of having more overhead as the number of calls increase. What do you think?